browser/src/neovim/NeovimInstance.ts

```diff
@@ -482,27 +497,29 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
             // Workaround for bug in neovim/node-client
             // The 'uiAttach' method overrides the new 'nvim_ui_attach' method
             Performance.startMeasure("NeovimInstance.Start.Attach")
-            return this._attachUI(size.cols, size.rows).then(
-                async () => {
-                    Log.info("Attach success")
-                    Performance.endMeasure("NeovimInstance.Start.Attach")
-
-                    // TODO: #702 - Batch these calls via `nvim_call_atomic`
-                    // Override completeopt so Oni works correctly with external popupmenu
-                    // await this.command("set completeopt=longest,menu")
-
-                    // set title after attaching listeners so we can get the initial title
-                    await this.command("set title")
-
-                    Performance.endMeasure("NeovimInstance.Start")
-
-                    this._initComplete = true
-                },
-                (err: any) => {
-                    this._onError(err)
-                    this._initComplete = true
-                },
-            )
+            try {
+                await this._attachUI(version, size.cols, size.rows)
+
+                Log.info("Attach success")
+                Performance.endMeasure("NeovimInstance.Start.Attach")
+
+                if (startUpErrorsChecked === false) {
+                    await this._checkAndFixIfBlocked()
+                }
```

Comment:
Not needed for Nvim 0.3.2, because `--embed` (without `--headless`) waits for the UI to attach before showing a "Press Enter" prompt. Therefore `_checkAndFixIfBlocked` is not needed. Any "Press Enter" prompt is normal prompt that should have been displayed to the user by now.